vra/resource_fabric_network_test.go

```diff
@@ -0,0 +1,78 @@
+package vra
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/vmware/vra-sdk-go/pkg/client/fabric_network"
+)
+
+func TestAccVRAFabricNetworkVsphere_importBasic(t *testing.T) {
+	resourceName := "vra_fabric_network_vsphere.this"
+	fabricNetworkID := os.Getenv("VRA_FABRIC_NETWORK_VSPHERE_ID")
+	createErrorRegex := regexp.MustCompile("vra_fabric_network resources are only importable")
+
+	checkFn := func(s []*terraform.InstanceState) error {
+		if len(s) != 1 {
+			return fmt.Errorf("expected 1 state %#v", s)
+		}
+
+		fabricNetworkVsphereState := s[0]
+		if fabricNetworkID != fabricNetworkVsphereState.ID {
+			return fmt.Errorf("expected fabric network ID of %s,%s received instead", fabricNetworkID, fabricNetworkVsphereState.ID)
+		}
+		return nil
+	}
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckVsphere(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckVRAFabricNetworkVsphereDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccCheckVRAFabricNetworkVsphereConfig(),
+				ExpectError: createErrorRegex,
+			},
+			{
+				Config:           testAccCheckVRAFabricNetworkVsphereConfig(),
+				ResourceName:     resourceName,
+				ImportState:      true,
+				ImportStateId:    fabricNetworkID,
+				ImportStateCheck: checkFn,
+			},
+		},
+	})
+}
+
+func testAccCheckVRAFabricNetworkVsphereDestroy(s *terraform.State) error {
+	apiClient := testAccProviderVRA.Meta().(*Client).apiClient
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type == "vra_fabric_network_vsphere" {
+			_, err := apiClient.FabricNetwork.GetVsphereFabricNetwork(fabric_network.NewGetVsphereFabricNetworkParams().WithID(rs.Primary.ID))
+			if err == nil {
+				return fmt.Errorf("Resource 'vra_fabric_network_vsphere' still exists with id %s", rs.Primary.ID)
```

Comment:
As the resource destroy doesn't delete the fabric network in vRA, should this be read as "Resource 'vra_fabric_network_vsphere' **doesn't exist** with id %s"?